locale/lang-ca.js

```diff
@@ -555,14 +565,22 @@ SnapTranslator.dict.ca = {
         'mode turbo?',
     'set turbo mode to %b':
         'posa el mode turbo a %b',
-
+    'is %setting on?':
+        'paràmetre %setting actiu?',
+    'set %setting to %b':
+        'fixa el paràmetre %setting a %b',
+    'turbo mode':
+        'mode turbo',
+    'flat line ends':
+        'finals de línia plans',
```

Comment:
Maybe `rectes` is clearer?

Reply:
Oh, I see you changed `rectes` to `plans` everywhere. Whatever works :)

(I think `rectes` is clearer because we're talking about how the pen line ends look though, but `plans` works too)